simple_history/models.py

```diff
@@ -448,19 +455,17 @@ def diff_against(self, old_history):
 
         changes = []
         changed_fields = []
-        instance_fields = [field.name for field in self.instance._meta.fields]
-        old_instance_fields = [
-            field.name for field in old_history.instance._meta.fields
-        ]
-        for field in self._meta.fields:
-            if field.name in instance_fields and \
-               field.name in old_instance_fields:
-                old_value = getattr(old_history, field.name, '')
-                new_value = getattr(self, field.name)
+        old_values = model_to_dict(old_history.instance)
+        current_values = model_to_dict(self.instance)
+        for field in current_values.keys():
+            if field in current_values and \
```

Comment:
I realize the equivalent conditional existed in the previous implementation, but I think it's unnecessary. `field` is coming from `current_values.keys()` so this will always evaluate to `True`. Can we drop it?